ui/admin/app/controllers/application.js

```diff
@@ -7,6 +7,7 @@ import Controller from '@ember/controller';
 import { service } from '@ember/service';
 import { getOwner } from '@ember/application';
 import { action } from '@ember/object';
+import { defaultValidator } from 'ember-a11y-refocus';
```

Comment:
Should we actually import this rather that depending it being available from a dependency?

Reply:
good point! Ill add `ember-ally-refocus` as a dep. in admin ui